roles/postgres_upgrade/tasks/configure.yml

```diff
@@ -0,0 +1,36 @@
+---
+
+- name: disable old database auto-start
+  lineinfile:
+    dest: "/etc/postgresql/{{ pg_old }}/main/start.conf"
+    regexp: "^auto"
+    line: "manual"
+  become: yes
+
+- name: set new cluster port to 5432
+  lineinfile:
+    dest: "/etc/postgresql/{{ pg_new }}/main/postgresql.conf"
```

Comment:
I'm not sure I'm following. When do we install the new PostgreSQL version and so this path gets created? I don't see it.

Reply:
The role tasks are split up a bit into sections. `roles/postgres_upgrade/tasks/install.yml ` is run before this.